.github/workflows/deploy.yml

```
@@ -11,10 +11,52 @@ jobs:
+          docker logout
+
+      - name: ssh into ec2 and deploy
+        uses: appleboy/ssh-action@v1
+        with:
+          host: ${{ secrets.EC2_HOST }}
+          username: ${{ secrets.EC2_USERNAME }}
+          key: ${{ secrets.EC2_PRIVATE_KEY }}
+          script: |
+            DOCKER_IMAGE="${{ secrets.DOCKER_USERNAME }}/${{ secrets.DOCKER_REPOSITORY_NAME }}"
+            EC2_ENV_DIR="/home/${{ secrets.EC2_USERNAME }}/HGDGDS-Node"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)